appengine/src/main/java/com/google/googleinterns/gscribe/modules/DaoModule.java

```diff
@@ -40,4 +40,27 @@ public ExamMetadataDao examMetadataDaoProvider(DBI dbi) {
         return dbi.onDemand(ExamMetadataDao.class);
     }
 
+    @Inject
+    @Provides
+    @Singleton
+    public QuestionsDao examDaoProvider(DBI dbi, ObjectMapper objectMapper) {
+        dbi.registerMapper(new QuestionsDao.QuestionsMapper(objectMapper));
```

Comment:
You can inject the QuestionMapper as well, just have an Inject constructor.

Reply:
Added a new MapperModule for this.